modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -9,7 +9,7 @@ Bonding enables multiple network interfaces to be aggregated into a single logic
 
 Bond-CNI can be created using SR-IOV virtual functions and placing them in the container network namespace.
 
-{product-title} only supports Bond-CNI using SR-IOV virtual functions. Physical interfaces are not supported.
+{product-title} only supports Bond-CNI using SR-IOV virtual functions, managed by SR-IOV CNI. Other CNI or types of interfaces are not supported.
```

Comment:
From reading this line, I'm not certain if the Bond-CNI is managed by SR-IOV CNI or if the virtual functions are. I don't know enough about the topic to determine this without more information. To enhance localisation and remove any potential ambiguity, you could split the "managed by SR-IOV CN" aspect into a separate sentence and provide an explicit statement about what must be managed by the SR-IOV CNI. In the following example, you could replace `<component>` with whatever it is that is managed:

> {product-title} only supports Bond-CNI using SR-IOV virtual functions. The `<component>` must be managed by SR-IOV CNI.